src/content/guides/development.md

```diff
@@ -394,10 +395,9 @@ When using automatic compilation of your code, you could run into issues when sa
 
 To disable this feature in some common editors, see the list below:
 
-* **Sublime Text 3** - Add `atomic_save: "false"` to your user preferences.
-* **IntelliJ** - use search in the preferences to find "safe write" and disable it.
-* **Vim** - add `:set backupcopy=yes` to your settings.
-* **WebStorm** - uncheck Use `"safe write"` in `Preferences > Appearance & Behavior > System Settings`.
+* **Sublime Text 3**: Add `atomic_save: "false"` to your user preferences.
```

Comment:
I think we can remove the version "3". WDYT?

Reply:
No, devs use all kind of tools

Reply:
I meant to remove the version number only. Not Sublime Text. :sweat_smile:

Reply:
No, that option could no exists in previous versions, it is better to be explicit.